stream-transformation-tool-api/src/main/java/uk/gov/justice/tools/eventsourcing/transformation/api/EventTransformation.java

```diff
@@ -17,15 +21,25 @@
      * @param event - the event to check
      * @return TRUE if the event is eligible to have the transformation applied to it.
```

Comment:
should probably update javadoc to mention default implementation is false also

Reply:
Done